D/DASSL/Compat.toml

```diff
@@ -33,5 +33,5 @@ DiffEqBase = "4-5"
 ["2.2-2"]
 julia = "1"
 
-["2.3-2"]
+["2.3"]
 DiffEqBase = "5"
```

Comment:
Does this version not require DiffEqBase 5?

Reply:
it probably does